### PR TITLE
Update main_lastman-standing-app.yml

### PR DESCRIPTION
**Pull request by CaoimhinHawk1**
Updating workflow to succesfully build and deploy our java web-application to Docker_hub for testing purposes

### CODE REVIEW

**Comment by olanhealy** [comment]
can this be deleted?

File: LMS/src/main/java/com/example/LMS/LmsApplication.java
Comment (olanhealy) [inline LMS/src/main/java/com/example/LMS/LmsApplication.java:1]: this class  has been deleted

**Review by olanhealy — COMMENTED** [review]
Mosly fine, except the LMS class has been deleted no longer in main

**Comment by CaoimhinHawk1** [comment]
Sorry about that, didnt check, ill delete it there when i get home

**Review by Whomstt — APPROVED** [review]
Looks good to me. Just the LmsApplication.java file is no longer in use as mentioned by Olan.

**Review by olanhealy — APPROVED** [review]
LGTM

**Pull request merged**